boudicca.base/common-model/src/jvmTest/kotlin/base/boudicca/keyfilters/KeySelectorTest.kt

```diff
@@ -18,8 +19,8 @@ class KeySelectorTest {
             .build()
             .selectSingle(testEntry())
 
-        assertTrue(result.isPresent)
-        assertEquals("description", result.get().first.toKeyString())
+        assertThat(result).isNotNull()
+        assertEquals("description", result!!.first.toKeyString())
```

Comment:
before we merge we should replace all those assertEquals as well with assertThat(...).isEqualTo() and then remove assertj and replace it entirely with assertk